driver/services/compute_test.go

```diff
@@ -100,9 +99,9 @@ func TestClient_CreateSecurityGroup(t *testing.T) {
 	sg, err := client.CreateSecurityGroup(sgName, 22)
```

Comment:
Don't we need here any defer?

Reply:
Not really, there is no `require.` calls after that, so `assert.NoError(t, client.DeleteSecurityGroup(sg.ID))` will be executed any way